src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -369,15 +368,14 @@ protected Task TestSmartTagGlyphTagsAsync(string initialMarkup, ImmutableArray<s
         {
             return TestAsync(
                 initialMarkup,
-                expectedMarkup, priority,
+                expectedMarkup,
                 new TestParameters(
-                    parseOptions, compilationOptions, options, fixProviderData, index));
+                    parseOptions, compilationOptions, options, fixProviderData, index, priority));
         }
 
         private async Task TestAsync(
             string initialMarkup,
             string expectedMarkup,
-            CodeActionPriority? priority,
```

Comment:
This is the core unused parameter that was removed. This method uses `TestParameters.priority` field for validation.